src/boot/index.js

```diff
@@ -11,20 +11,24 @@
 
 import { parseJsonConfig } from './parse-json-config';
 
-import boot from './boot';
+import { bootHypothesisClient, bootSidebarApp } from './boot';
 import processUrlTemplate from './url-template';
 import { isBrowserSupported } from './browser-check';
 
 if (isBrowserSupported()) {
   const settings = parseJsonConfig(document);
-  boot(document, {
-    assetRoot: processUrlTemplate(settings.assetRoot || '__ASSET_ROOT__'),
-    // @ts-ignore - `__MANIFEST__` is injected by the build script
-    manifest: __MANIFEST__,
-    sidebarAppUrl: processUrlTemplate(
+  const assetRoot = processUrlTemplate(settings.assetRoot || '__ASSET_ROOT__');
+  // @ts-ignore - `__MANIFEST__` is injected by the build script
+  const manifest = __MANIFEST__;
+
```

Comment:
Some commenting here would be helpful about what the following code does, in what order, and why.